### DISCUSSION TITLE
[Script request] Flame (Startpage/Homepage)

### DISCUSSION THREAD

**Discussion by chris400** [community-scripts/ProxmoxVE#115] [Request script] [answered]
### Application Name

Flame

### Website

https://github.com/pawelmalak/flame

### Description

I like Flame bc it is very easy to use and does _not_ require any config file editing.

Official desc reads:
_Flame is very easy to setup and use. With built-in editors, it allows you to setup your very own application hub in no time - no file editing necessary._

https://github.com/pawelmalak/flame/wiki/Installation-without-docker 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
There have been no more commits to this repository for over a year. I would say it is dead. I don't think it makes sense to make a script for it.

@community-scripts/contributor do you agree with me?

**Reply by Mellowlynx** [reply]
I agree, looks dead indeed, not smart to add or even use as security issues are not solved either.

**Comment by drego85** [comment]
Hi, here is the supported Flame repository: https://github.com/fdarveau/flame

**Reply by Mellowlynx** [reply]
Your point? Look at the dates; it's still dead.
<img width="1032" height="1186" alt="image" src="https://github.com/user-attachments/assets/53cd755a-65f0-4494-a440-8c0e6f130eaf" />

**Reply by drego85** [reply]
Hi, you’re right. I was misled by the date of the last commit shown on the homepage (February 3, 2026). However, looking at the [commit history](https://github.com/fdarveau/flame/commits/master/), I can see that some changes have been made, although they’re not substantial.

**Comment by chris400** [comment]
UPDATE:
It has been added https://github.com/community-scripts/ProxmoxVE/pull/15464 🥳

There was an update of the app in April https://github.com/pawelmalak/flame/releases/tag/v2.4.0

> - Fixed high severity authentication security issues (https://github.com/pawelmalak/flame/issues/465)
>
> Hi! Thank you for using Flame all these years.
> 
> It’s been almost three years since the last release, and I’ve essentially abandoned this and my other projects due to some life events - but I think I’m in a place to come back to it.
> 
> As for the future of this project, I’m considering releasing version 3.0 within the next month or two, featuring a modern stack, performance optimizations, and new features. I’m not making any promises about a specific date, but it’s coming. Stay tuned!
